File: Selenium/Project/crm_activity3.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
driver=webdriver.Chrome()
 
    # Navigate to the URL
driver.get("http://alchemy.hguy.co/crm")
#find the first copyright text in the footer.
copyRight =driver.find_element(By.ID,"admin_options")
		#print the text to the console.
print( "First copyright text in the footer is:" , copyRight.text)

File: Selenium/Project/CRM_Activity3.java
package crm_project;

import org.testng.annotations.Test;
import org.testng.annotations.BeforeClass;

import static org.testng.Assert.assertEquals;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.annotations.AfterClass;

public class CRM_Activity3 {

//declare WebDriver 
		WebDriver driver;
	  
	  
	  @BeforeClass
	  public void setUp() {
		  //initialize webdriver
		  driver = new ChromeDriver();
		  //open the page
		  driver.get("http://alchemy.hguy.co/crm");
	  }
	 
	  @Test
	  public void firstCopyTextTest() {
		  //find the first copyright text in the footer.
		WebElement copyRight=  driver.findElement(By.id("admin_options"));
		//print the text to the console.
		System.out.println( "First copyright text in the footer is:" + copyRight.getText());
		  assertEquals(copyRight.getText(), "© Supercharged by SuiteCRM");
	  }
	 

	  @AfterClass
	  public void tearDown() {
		  //quit the browser
		  driver.quit();
	  }


}
